dali/test/python/test_dali_tf_dataset_mnist_eager.py

```diff
@@ -12,8 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from test_dali_tf_dataset_mnist import *
 from nose_utils import raises
+from test_dali_tf_dataset_mnist import run_keras_single_device, skip_for_incompatible_tf, \
+    keras_model, get_dataset, EPOCHS, ITERATIONS, available_gpus, TARGET, clear_checkpoints, \
+    run_estimators_single_device, get_dataset_multi_gpu
```

Comment:
OMG. I really preferred `*` in such a short file.
Alternatively, how about `import test_dali_tf_dataset_mnist as mnist` and using qualified names?

Reply:
Went for `import test_dali_tf_dataset_mnist as mnist`